### PR TITLE
Add-storage-feedback

### PR DESCRIPTION
**Pull request by tsoudibi**

### CODE REVIEW

File: data/Budget.txt
Comment (limkongkiat) [inline data/Budget.txt:1]: Perhaps you could set git to ignore Budget.txt and FinancialList.txt?

**Review by limkongkiat — APPROVED** [review]
Overall LGTM, good work on extracting your messages as variables as well!

File: data/Budget.txt
Comment (tsoudibi) [reply data/Budget.txt:1]: i did, but if there is no Budget.txt and FinancialList.txt on github, the unit test checks done by github won't pass.

Comment (tsoudibi) [reply data/Budget.txt:1]: So I just added simple  Budget.txt and FinancialList.txt and added them into gitignor. So it won't update every time we commit

**Pull request merged**